# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/tests/unit/test_trust_anchors/ca2.pem
# Commit: 55e22304ccae
# Full Hash: 55e22304ccaea2af33219077000171821441822c
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2025-08-07 04:23:39
# Description:
#   Bug 1981271 - reset state specific to the built candidate chain root when continuing path building r=nkulatova
#   
#   Any state that NSSCertDBTrustDomain keeps with regard to a built candidate
#   chain (and in particular, the root of that chain), must be reset if path
#   building continues with other potential chains. In particular, this means that
# ==============================================================================

diff -r 2c224e8a6135 -r 55e22304ccae security/manager/ssl/tests/unit/test_trust_anchors/ca2.pem
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/security/manager/ssl/tests/unit/test_trust_anchors/ca2.pem	Wed Aug 06 18:13:58 2025 +0000
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
+MIICyzCCAbOgAwIBAgIUd1Ed4FZFnJiTG7iE/C/1JQgsvJEwDQYJKoZIhvcNAQEL
+BQAwDjEMMAoGA1UEAwwDY2EyMCIYDzIwMjAwMTAxMDAwMDAwWhgPMjA1MTAxMDEw
+MDAwMDBaMA4xDDAKBgNVBAMMA2NhMjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
+AQoCggEBALqIUahEjhbWQf1utogGNhA9PBPZ6uQ1SrTs9WhXbCR7wcclqODYH72x
+nAabbhqG8mvir1p1a2pkcQh6pVqnRYf3HNUknAJ+zUP8HmnQOCApk6sgw0nk27lM
+wmtsDu0Vgg/xfq1pGrHTAjqLKkHup3DgDw2N/WYLK7AkkqR9uYhheZCxV5A90jvF
+4LhIH6g304hD7ycW2FW3ZlqqfgKQLzp7EIAGJMwcbJetlmFbt+KWEsB1MaMMkd20
+yvf8rR0l0wnvuRcOp2jhs3svIm9p47SKlWEd7ibWJZ2rkQhONsscJAQsvxaLL+Xx
+j5kXMbiz/kkj+nJRxDHVA6zaGAo17Y0CAwEAAaMdMBswDAYDVR0TBAUwAwEB/zAL
+BgNVHQ8EBAMCAQYwDQYJKoZIhvcNAQELBQADggEBALXntDXj7ddTytFXp68wW234
+mtQ12gI/Ij1l7O8C6NbHm6FoFH0lbRBmfKBIIIWKprKo3HwZDU4iSWfgT6aaEHzR
+IFsnP4HAa4QjYXxL62qUZY8e80ni/p3HZ8NKJjadufa13kt4bfrORtawpumFiBkB
+xymzHMCN8ARsEm70UZIBihpKSOobDLmRJ1KCuJLsbMWHvDa9MQKn41x7A4pNYpx8
+zNn2aSYbhXtKvBenSkDenikmj+NLXtEpHsPfl2K+nHYlC428xWpmxcSPb7rj+W4N
+qPtpBeM7iUkq782z0hRWXO5ol1tOsPrcyKlvcM8H7Jij4A65eWkkxXMiy0cutgw=
+-----END CERTIFICATE-----